### PR TITLE
Create Release Notes.md

### PR DESCRIPTION
**Pull request by rgralakosi**
Contains the edgecmd v1.1 release notes.

### CODE REVIEW

File: V1.1/Release Notes.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
+uid: releaseNotes
+---
+
+# Release Notes
+
+EdgeCmd v1.1<br>
```
[the remark this answers is not in the document]
Comment (rgralakosi) [reply V1.1/Release Notes.md:7]: I don't think that will matter unless "utility" is capitalized. However, the utility really should probably be lower case "edgecmd".

File: V1.1/Release Notes.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+© 2020 OSIsoft, LLC. All rights reserved.
```
[the remark this answers is not in the document]
Comment (Xiaoli-OSI) [reply V1.1/Release Notes.md:8]: If not removed, it should be 2019-2020.

[the remark this answers is not in the document]
Comment (rgralakosi) [reply V1.1/Release Notes.md:8]: Okay, thanks!

File: V1.1/Release Notes.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+## Fixes and enhancements
+
+### Fixes
+
+The command line syntax and processing of edgecmd v1.1 has been entirely reworked from scratch, so there are no specific fixes.
+
+### Enhancements
+
+Changes from version 1.0 (Mayflower to Atlantis)
```
[the remark this answers is not in the document]
Comment (Xiaoli-OSI) [reply V1.1/Release Notes.md:24]: Program names are for internal use, we should remove them.

File: V1.1/Release Notes.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+# Release Notes
+
+EdgeCmd v1.1<br>
+© 2020 OSIsoft, LLC. All rights reserved.
+
+## Overview
+
+Edge Data Store (EDS) and PI Adapters have a REST API that allows user interaction. Configuration is one of the most common interactions. This command line tool, edgecmd, uses this REST API to apply all changes and administration functions to EDS and PI adapters. Specifically, the edgecmd's command line interface uses configuration and administration functions of the REST API, so the user does not need to know the REST routes. Edgecmd thus improves the user experience, facilitates easier scripting experiences and provides built-in help. 
+
+For more information see edgecmd overview.
```
[the remark this answers is not in the document]
Comment (rgralakosi) [reply V1.1/Release Notes.md:14]: Yes, it should. I made the change. Thanks!

File: V1.1/Release Notes.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (rgralakosi) [reply V1.1/Release Notes.md:24]: Good catch! I removed that!

File: V1.1/Release Notes.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (rgralakosi) [reply V1.1/Release Notes.md:8]: I removed the copyright. The change is on the branch now so I am Resolving this comment.

[the remark this answers is not in the document]
Comment (Xiaoli-OSI) [reply V1.1/Release Notes.md:8]: @TinaUll , could you confirm, or add it back if it is required?

[the remark this answers is not in the document]
Comment (TinaUll) [reply V1.1/Release Notes.md]: @Xiaoli-OSI I will add it back if required

**Comment by TinaUll** [comment]
Closing this one - moved the content to the V1.2 folder in the [EdgeCmd overhaul](https://github.com/osisoft/Edgecmd-Docs/pull/11) branch and renamed the file accordingly.

**Pull request closed without merging**